packages/pubsub/src/Providers/MqttOverWSProvider.ts

```diff
@@ -125,16 +125,37 @@ export class MqttOverWSProvider extends AbstractPubSubProvider {
 
     protected _topicObservers: Map<string, Set<ZenObservable.SubscriptionObserver<any>>> = new Map();
 
+    public mqttTopicMatch(filter: string, topic: string) {
```

Comment:
```suggestion
    export function mqttTopicMatch(filter: string, topic: string) {
```
Sorry about that, when I saw the unit test it doesn't make sense for `mqttTopicMatch` to be a method of the class. Is just an static function that needs to be exported (it doesn't depend of MqttProvider). You will need to move this function outside of the class.